src/frameworks/angular/utils.ts

```diff
@@ -10,6 +10,7 @@ import { BUILD_TARGET_PURPOSE } from "../interfaces";
 import { AssertionError } from "assert";
 import { assertIsString } from "../../utils";
 import { coerce } from "semver";
+import { JsonObject } from "@angular-devkit/core";
```

Comment:
change it to `import type` and move to the beginning of the file as others Angular imports are